Assignment3-Loops-Siffrinn/src/assignment3/loops/siffrinn/Assignment3LoopsSiffrinn.java

```diff
@@ -0,0 +1,38 @@
+package assignment3.loops.siffrinn;
+
+import java.util.Scanner;   //needed for the Scanner class
+
+//This program will show the reverse of a word
+
+public class Assignment3LoopsSiffrinn {
+  
+    public static void main(String[] args) {
+ 
+        String origWord;    //original word
+        
+       Scanner keyboard = new Scanner(System.in); //initialize the scanner
+        
+        //Let's user choose a word and stores it in the variable origWord
+        System.out.println("Pick a word with at least 5 letters.\n");
+        origWord = keyboard.nextLine();
```

Comment:
Thanks, @ordizzle, When I was reviewing the book, I do not recall .next() being able to be used alone. Can next() be used as a generic scanner class method for anything? Everything I saw was nextLine, nextInt, nextDouble, nextLong, etc. I will look it up, just thought you could shed some light since you mentioned it above. Thanks.